spreadsheet_dashboard_oca/views/spreadsheet_dashboard.xml

```diff
@@ -15,21 +15,42 @@
         <field name="arch" type="xml">
             <tree position="attributes">
                 <attribute name="create">1</attribute>
+                <attribute name="decoration-muted">not active</attribute>
             </tree>
             <tree position="inside">
+                <field name="can_edit" invisible="1" />
                 <field
                     name="data"
                     groups="base.group_no_one"
                     widget="binary"
                     filename="name"
                 />
+                <!-- Put the button 2 times for having the edit button explicitly disabled for discoverability -->
                 <button
                     name="open_spreadsheet"
                     type="object"
                     string="Edit"
                     icon="fa-pencil"
+                    disabled="1"
                     groups="base.group_system"
+                    attrs="{'invisible': [('can_edit', '=', True)]}"
                 />
+                <button
+                    name="open_spreadsheet"
+                    type="object"
+                    string="Edit"
+                    icon="fa-pencil"
+                    groups="base.group_system"
+                    attrs="{'invisible': [('can_edit', '=', False)]}"
+                />
+                <button
+                    name="copy"
+                    type="object"
+                    string="Copy"
+                    icon="fa-copy"
+                    groups="base.group_system"
```

Comment:
Only setting users??

Reply:
I haven't changed this. Only be consistent, but yes, it seems a bit restricted. What do you think, @etobella ?

Reply:
It was done this way in order to set some groups, but we can create a new set if we think that it is the best option.